pkg/dbutil/common.go

```diff
@@ -208,6 +213,52 @@ func GetRandomValues(ctx context.Context, db *sql.DB, schemaName, table, column
 	return randomValue, nil
 }
 
+// GetMinMaxValue return min and max value of given column by specified limitRange condition.
+func GetMinMaxValue(ctx context.Context, db *sql.DB, schema, table, column string, limitRange string, collation string, args []interface{}) (string, string, error) {
```

Comment:
is `args` needed here?

Reply:
limitRange may looks like "a > ? AND a < ?", need use args when query